fansi/shared/src/test/scala/fansi/FansiTests.scala

```diff
@@ -170,16 +170,42 @@ object FansiTests extends TestSuite{
       }
     }
     def tabulate(all: Seq[fansi.Attr]) = {
-      all.map(attr => attr.toString + " " * (25 - attr.name.length))
+      all.map(attr => attr.toString + " " * (30 - attr.name.length))
          .grouped(3)
          .map(_.mkString)
          .mkString("\n")
     }
 
+    def square(all : Seq[fansi.Attr]) = {
+      all.map( attr => attr.escapeOpt.getOrElse("") + "#")
+      .grouped(32)
+      .map(_.mkString)
+      .mkString("\n")
+    }
+
 
     'colors - tabulate(fansi.Color.all)
+
     'backgrounds - tabulate(fansi.Back.all)
 
+    'red - fansi.Color.True(255,0,0)
+
+    'redhexa - fansi.Color.True(0xFF0000)
+
+    'green - fansi.Color.True(0,255,0)
+
+    'greenhexa - fansi.Color.True(0x00FF00)
+
+    'blue - fansi.Color.True(0,0,255)
+
+    'bluehaxe - fansi.Color.True(0x0000FF)
```

Comment:
Can we group these under a `'trueColorConstructors` test group? Doesn't make sense to have them top-level like the `'trueColors` and `'colors` groups